modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/TranslateDisplayContext.java

```diff
@@ -133,6 +136,69 @@ else if (infoFieldSetEntry instanceof InfoFieldSet) {
 		return _infoForm.getInfoFieldSetEntries();
 	}
 
+	public HashMap<String, Object> getInfoFieldSetEntriesData() {
```

Comment:
In method signatures, we should always use interfaces, not impl classes. In this case, use `Map<String, Object>` instead of HashMap.